cmd/globals.go

```diff
@@ -96,6 +99,12 @@ var (
 	// IsSSL indicates if the server is configured with SSL.
 	globalIsSSL bool
 
+	globalTLSCertificate *tls.Certificate
+
+	globalHTTPServer        *miniohttp.Server
+	globalHTTPServerErrorCh = make(chan error)
+	globalOSSignalCh        = make(chan os.Signal, 1)
```

Comment:
Once the signal.Notify gets control we should also close the signal handler such that signal.Stop is called.  This is achieved in signals.go . Not sure why keeping it global helps.. it can still be allocated inside the signalTrap and returned.

All channels would still receive since kernel signal handler is common no matter how many signal.Notify you invoke.